components/Ripple/src/MDCRippleTouchController.m

```diff
@@ -15,7 +15,28 @@
 #import "MDCRippleTouchController.h"
 
 @implementation MDCRippleTouchController {
+  MDCRippleView *_rippleView;
```

Comment:
Nit: Why is this being redeclared as an explicit ivar rather than an `@synthesize`? Does it have side effects different from an iVar?  

I ask because in our code when I see an ivar directly allocated in the implementation, I interpret it as being fully-internal. When I see an `@synthesize` directive, I read it as storing the value for a public property.